rest-api-spec/src/main/resources/rest-api-spec/api/search.json

```
@@ -182,8 +182,8 @@
         "description":"Whether to calculate and return scores even if they are not used for sorting"
       },
       "track_total_hits":{
-        "type":"boolean",
-        "description":"Indicate if the number of documents that match the query should be tracked"
+        "type":"boolean|number",
```

```suggestion
        "type":"boolean|long",
```

This should be `long`, in line with https://github.com/elastic/elasticsearch/tree/master/rest-api-spec#type-definition